api/routes.go

```diff
@@ -40,6 +41,7 @@ func (s *server) routes() {
 	api.HandleFunc("/{account}/clusters/{cluster}/taskdefs/{taskdef}/tasks", s.TaskDefRunHandler).Methods(http.MethodPost)
 	api.HandleFunc("/{account}/clusters/{cluster}/taskdefs/{taskdef}/tasks", s.TaskDefTaskListHandler).Methods(http.MethodGet)
 	api.HandleFunc("/{account}/clusters/{cluster}/taskdefs/{taskdef}/tasks/{task}", s.TaskShowHandler).Methods(http.MethodGet)
+	api.HandleFunc("/{account}/clusters/{cluster}/taskdefs/{taskdef}/tasks/{task}", s.TaskStopHandler).Methods(http.MethodDelete)
```

Comment:
Seems a little weird using DELETE to stop a task. Is it basically gone after you stop it? If we would never restart it, then I guess it's fine.

Reply:
yeah these are one and done 👍